src/buildTree.js

```diff
@@ -43,4 +38,19 @@ const buildTree = function(
     }
 };
 
+export const buildHTMLTree = function(
+    tree: ParseNode[],
+    expression: string,
+    settings: Settings,
+): domTree.span {
+    const options = optionsFromSettings(settings);
+    const htmlNode = buildHTML(tree, options);
+    const katexNode = buildCommon.makeSpan(["katex"], [htmlNode]);
```

Comment:
Is the katexNode necessary here? Why not just return `htmlNode`?

Reply:
Hmm, good question.  `htmlNode` would be natural too.  My reasoning here was that there are various CSS rules that only apply with the `.katex-display .katex` nesting...

Reply:
Ah okay.